ios/Plugin.xcodeproj/project.pbxproj

```diff
@@ -190,8 +190,9 @@
 		50ADFF7F201F53D600D50D53 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
+				BuildIndependentTargetsInParallel = YES;
```

Comment:
this was suggested by Xcode. It doesn't seem to pose any problems.

Reply:
that's fine